ts/src/header-validator/source.test.ts

```diff
@@ -1927,6 +1930,223 @@ const testCases: TestCase[] = [
       },
     ],
   },
+
+  // Attribution Scope
+  {
+    name: 'attribution-scope-limit-negative',
+    json: `{
+      "destination": "https://a.test",
+      "trigger_data": [],
```

Comment:
Is this field relevant for this test case? Same for all the other new test cases below.

Reply:
Removed.

Reply:
Just uploaded a new patch.